### PR TITLE
Format examples using Prettier

### PR DESCRIPTION
**Pull request by bsmth**
This PR takes a sweep across all webaudio examples using Prettier.

Ignore minified asset `violent-theremin/scripts/respond.js`

Limiting the changes to 10 files so that this is digestible in a single PR.

### CODE REVIEW

File: audioworklet/index.html [hunk 1]
```diff
@@ -2,39 +2,56 @@
...
+
+    <button id="toggle">Toggle Sound</button>
+
     <div class="control-box">
       <div class="control">
-        <span class="control-label"><label for="osc-gain">Oscillator gain:</label></span>
-        <input type="range" id="osc-gain" name="osc-gain" min="0.0" max="1.0" step="0.05" value="0.1">
+        <span class="control-label"
+          ><label for="osc-gain">Oscillator gain:</label></span
+        >
```
Comment (dipikabh) [inline audioworklet/index.html:29]: This looks odd

File: audioworklet/index.html [hunk 2]
```diff
@@ -2,39 +2,56 @@
...
+        />
       </div>
-      
+
       <div class="control">
-        <span class="control-label"><label for="hiss-gain">Hiss gain:</label></span>
-        <input type="range" id="hiss-gain" name="hiss-gain" min="0.0" max="1.0" step="0.05" value="0.2">
+        <span class="control-label"
+          ><label for="hiss-gain">Hiss gain:</label></span
+        >
```
Comment (dipikabh) [inline audioworklet/index.html:44]: This is similar to the previous one

**Review by dipikabh — COMMENTED** [review]
Looks good. Not sure if the two instances I pointed out are how they should be.

File: audioworklet/index.html (hunk 1 above)
Comment (bsmth) [reply audioworklet/index.html]: tnx, fixed in https://github.com/mdn/webaudio-examples/pull/88/commits/6a9d10bd9fcd0ae911cbd22f6139f0fb543c7e9e****

File: audioworklet/index.html (hunk 2 above)
Comment (bsmth) [reply audioworklet/index.html]: tnx, fixed in https://github.com/mdn/webaudio-examples/commit/6a9d10bd9fcd0ae911cbd22f6139f0fb543c7e9e

**Comment by bsmth** [comment]
Thanks, Dipika 👍🏻

**Pull request merged**